neonvm/README.md

```diff
@@ -68,6 +70,12 @@ root@neonvm:~# curl -k https://kubernetes/version
 <press CTRL-a k to exit screen session>
 ```
 
+#### SSH
+
+```sh
+kubectl exec -it $(kubectl get neonvm vm-debian -ojsonpath='{.status.podName}') -- screen /dev/pts/0
```

Comment:
I think you want to do ssh here

Reply:
Why do we even have two very similar sections in this doc.

Reply:
fixed in cad4d78e271c58aa5fad51715780e0b707d6c83a. I'll make a separate issue for your second comment.